conf/modules.config

```diff
@@ -928,25 +928,27 @@ process{
 // PAIR_VARIANT_CALLING
 
     //ASCAT
-    if (params.tools && params.tools.contains('ascat')) {
-        withName: 'ASCAT' {
-            ext.args = {
-                [
-                "gender": meta.sex,
-                //"genomeVersion": "hg19"
-                //"purity": (optional),
-                //"ploidy": params.ploidy,
-                //"gc_files": (optional),
-                //"minCounts": (optional),
-                //"chrom_names": (optional),
-                //"min_base_qual": (optional),
-                //"min_map_qual": (optional),
-                //"ref_fasta": (optional),
-                //"skip_allele_counting_tumour": (optional),
-                //"skip_allele_counting_normal": (optional)
-                ]
-            }
-        }
+    withName: 'ASCAT' {
+
+        ext.args = {[
+            "gender": meta.gender,
+            "genomeVersion": params.ascat_genome,
+            "purity": params.ascat_purity,
+            "ploidy": params.ploidy,
+            //"minCounts": (optional),
+            "chrom_names": 'c("21","22")' //,
```

Comment:
I assume we want all chromosomes here?

Reply:
Since we only have human reference files can stick to humans and leave instructions on how to use other chromosomes there

Reply:
Good point, I think ascat can really only be used for humans. 
But I'll add a chromosome parameter, so that the chromosomes of interest can be specified when running ascat.